src/java/org/apache/cassandra/audit/AuditLogManager.java

```diff
@@ -197,8 +198,38 @@ public void logBatch(String batchTypeName, List<Object> queryOrIdList, List<List
         }
     }
 
-    public void reloadFilters()
+    public synchronized void disableAuditLog()
+    {
+        logger.info("Audit logging is disabled, stopping any existing loggers"); //debug
+        if (this.auditLogger != null)
+        {
+            this.auditLogger.stop();
+            this.auditLogger = null;
+        }
+    }
+
+    public synchronized void enableAuditLog(AuditLogOptions auditLogOptions) throws ConfigurationException
     {
-        filter = AuditLogFilter.create();
+        logger.trace("Audit logging is being enabled. Reloading AuditLogOptions.");
```

Comment:
nit: probably should be at info.

Reply:
Might be too much at info, will keep at debug. We already have similar statement in StorageService